src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -49,6 +49,16 @@ public without sharing class UTIL_CustomSettingsFacade {
     static List<Relationship_Auto_Create__c> relationshipAutocreate;
     static List<Relationship_Lookup__c> relationshipLookup;
 
+    // Constructor
+    public UTIL_CustomSettingsFacade() {
```

Comment:
Wait, weren't we moving these methods to its own class? I thought it was going to happen when you do the permission check in a separate branch. But I see the permission check is happening here.